### PR TITLE
ADD users/your_emojis.md

### PR DESCRIPTION
**Pull request by OrionVi1998**

### CODE REVIEW

File: src/users/your_emojis.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+In this section we'll explain the first two sources while you can learn more about the other two in their respective sections.
+
+## Common Server Emojis
+
+The first source of emojis are the Common Server Emojis.
+These are the emojis that have been directly uploaded onto discord servers where both you and the bot are.
+This emulates the behaviour of nitro closely as we can see in the following example:
+
+Imagine an emoji named `:nqn-nqn:` has been uploaded to server A.
```
Comment (muddyfish) [inline src/users/your_emojis.md]: Emojis can't have a dash in them. I'd say `:nqn:` instead

Comment (muddyfish) [inline src/users/your_emojis.md]: I'd prefer something like 'If there was an emoji named ... uploaded to server A'

File: src/users/your_emojis.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+Both you and the bot are in this server.
+This allows you to use the emojis of Server A on all the servers you share with the bot.
+
+You can see all the servers you share with the bot through the command `!mutuals`
```
Comment (muddyfish) [inline src/users/your_emojis.md]: Yes, but I think the `my_emotes` command is probably better to call out here, as it also includes a list of the emotes in those servers.

**Review by muddyfish — CHANGES_REQUESTED** [review]

File: src/users/your_emojis.md [hunk 3]
```diff
@@ -15,11 +15,11 @@ The first source of emojis are the Common Server Emojis.
...
 This emulates the behaviour of nitro closely as we can see in the following example:
 
-Imagine an emoji named `:nqn-nqn:` has been uploaded to server A.
+> An emoji named `:nqn:` has been uploaded to server A.
 Both you and the bot are in this server.
-This allows you to use the emojis of Server A on all the servers you share with the bot.
+This allows you to use the emojis of server A on all the servers you share with the bot.
 
-You can see all the servers you share with the bot through the command `!mutuals`
+You can see all the emotes you can use through the command `/my_emotes`
```
Comment (lorypelli) [inline src/users/your_emojis.md:22]: Do you prefer slash command or prefix command here?

Comment (muddyfish) [reply src/users/your_emojis.md:22]: Slash.

**Pull request merged**